prowler/providers/aws/services/iam/iam_password_policy_uppercase/iam_password_policy_uppercase_fixer.py

```diff
@@ -0,0 +1,50 @@
+from prowler.lib.logger import logger
+from prowler.providers.aws.services.iam.iam_client import iam_client
+
+
+def fixer(resource_id: str) -> bool:
+    """
+    Enable IAM password policy to require uppercase characters or the configurable value in prowler/config/config_fixer.yaml.
+    Requires the iam:UpdateAccountPasswordPolicy permission:
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": "iam:UpdateAccountPasswordPolicy",
+                "Resource": "*"
+            }
+        ]
+    }
+    Returns:
+        bool: True if IAM password policy is updated, False otherwise
+    """
+    try:
+        iam_client.client.update_account_password_policy(
```

Comment:
I don't get the point of doing the same in all these new fixers, if you just want to fix the `RequireUppercaseCharacters` you have to leave the rest of the values as the AWS account has configured. You have this values on the client so it wouldn't be difficult.

Reply:
Please add a test for this function.